unity-renderer/Assets/Rendering/ProceduralSkybox/ToolProceduralSkybox/Scripts/Editor/Procedural Skybox/Base Layers/RenderHorizonPlane.cs

```diff
@@ -0,0 +1,40 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+
+namespace DCL.Skybox
+{
+    public class RenderHorizonPlane
+    {
+        public static void RenderLayer(ref float timeOfTheDay, EditorToolMeasurements toolSize, SkyboxConfiguration config)
+        {
+            // Horizon Plane
+            RenderSimpleValues.RenderTexture("Texture", ref config.horizonPlaneTexture);
```

Comment:
if `RenderLayer()` is called frequently let's cache all these new strings into CONSTs. I even see some of them (like `"Texture"`, `"Tiling"`, `"Offset"`) repeated for example in `RenderBackgroundColorLayer.cs`, maybe those (or all) could be in a statically accessible place or config structure.